reconcile/test/utils/test_early_exit_cache.py

```diff
@@ -0,0 +1,185 @@
+import datetime
+from typing import Any
+from unittest.mock import create_autospec
+
+import pytest
+from deepdiff import DeepHash
+from pytest_mock import MockerFixture
+
+from reconcile.utils.early_exit_cache import (
+    CacheKey,
+    CacheStatus,
+    CacheValue,
+    EarlyExitCache,
+)
+from reconcile.utils.secret_reader import SecretReaderBase
+from reconcile.utils.state import State
+
+
+@pytest.fixture
+def state() -> Any:
```

Comment:
NIT: You can use the `mocker` fixture here:
```python
from pytest_mock import MockerFixture

@pytest.fixture
def state(mocker: MockerFixture) -> Any:
    return mocker.create_autospec(State)
```

Reply:
Is `mocker.create_autospec` just calling `unittest.mock.create_autospec`? Any advantage using `mocker` fixture?

Reply:
TBH, there is no real advantage. I find the `mocker` fixture just handy.

Reply:
Oh, my preference is only use `mocker.patch` to deal with dependencies need patching, for creating simple mocks/stubs I use `unitest.mock` builtin methods.